pyhidra/launcher.py

```diff
@@ -216,12 +213,18 @@ def start(self):
         for _, details in self._plugins:
             self._uninstall_old_plugin(details)
 
+        # Setup jpype Keyword Arguments
+        if 'classpath' in jpype_kwargs:
+            raise TypeError('classpath specified twice')
```

Comment:
Instead of erroring out, should we merge the self.class_path​ and the user provided classpath​ together?

Reply:
Sorry for the misunderstanding.  A simple multiprocessing call should do the trick. Otherwise, if it is getting too complex, not having this test isn't a big deal.

Reply:
ah yeah, that makes sense. Now merging.